.travis.yml

```diff
@@ -1,5 +1,6 @@
  language: scala
  scala:
    - "2.10.4"
+   - "2.11.5"
```

Comment:
@schleichardt the whole build chain is triggered two times: one time with Scala 2.10.4 and one time with Scala 2.11.5. See https://travis-ci.org/sphereio/sphere-scala-libs/builds/52184796 as an example.